src/main/java/List.java

```diff
@@ -9,16 +9,10 @@
     }
 
     public void queryItems(){
-        System.out.println("____________________________________________________________\n" +
-                "Here are the tasks in your list: ");
-        int count=1;
-        for(Task item: items){
-            if(item!=null) {
-                System.out.println(count+". "+item.toString());
-                count++;
-            } else {
-                break;
-            }
+        System.out.println("____________________________________________________________");
```

Comment:
Maybe you can use some methods to print the horizontal lines and other messages.